interceptors/Mementifier.cfc

```diff
@@ -121,9 +121,47 @@ component{
 				thisName = ( md.keyExists( "entityName" ) ? md.entityName : listLast( md.name, "." ) );
 			}
 
-			thisMemento.defaultIncludes = ormGetSessionFactory()
-				.getClassMetaData( thisName )
-				.getPropertyNames();
+			var ormUtil = server.coldfusion.productname == "ColdFusion Server" ? new cborm.models.util.CFORMUtil() : new cborm.models.util.LuceeORMUtil(); 
```

Comment:
if we are using cborm, why not just instantiate a BaseORM Service and call what you need from it?

Reply:
Replied.  I can add wirebox injection to Mementifier, though. It's your call.

Reply:
You can still use `new cborm.models.BaseORMService()`